src/BinSkim.Rules/PERules/BA2006.BuildWithSecureTools.cs

```diff
@@ -116,6 +116,13 @@ public override void AnalyzePortableExecutableAndPdb(BinaryAnalyzerContext conte
                 Symbol om = omView.Value;
                 ObjectModuleDetails omDetails = om.GetObjectModuleDetails();
 
+                if (omDetails.WellKnownCompiler != WellKnownCompilers.MicrosoftC
+                    && omDetails.WellKnownCompiler != WellKnownCompilers.MicrosoftCxx)
+                {
+                    // TODO: https://github.com/Microsoft/binskim/issues/114
```

Comment:
>TODO [](http://example.com/codeflow?start=23&length=4)

TODO: MikeFan (1/6/2022)
We need to take a step back and comprehensively review our compiler/language support.
[https://github.com/Microsoft/binskim/issues/114](https://github.com/Microsoft/binskim/issues/114)

Reply:
fixed